### ISSUE TITLE
Bug: LinkedIn Profile page results in HTTP2_STATUS_invalid (999 code)

### ISSUE THREAD

**Issue by GlenDC** [ulixee/secret-agent#451]
e.g. <https://www.linkedin.com/in/satyanadella/>

Something like <https://www.linkedin.com> does work.

**Comment by blakebyrnes** [comment]
Looks like LinkedIn is actually returning a 999 status code, which regular Chrome will still load if there's a response body. Automated Chrome throws a network failure and redirects to a chrome error page. Very weird behavior. I wonder which part of Chrome is intercepting and handling this where automation is not?

**Comment by GlenDC** [comment]
yes indeed. I wonder as well. LinkedIn is no target of me, but noticed it in some random test.

**Comment by blakebyrnes** [comment]
Actually, it looks like this is a nodejs issue. Digging through their source code, there's no way I can find to send a status code that isn't per-http spec. They reject > 599. Simple solution is to modify statuses above 599 to be an acceptable value... but that is definitely detectable if you were to send an XHR request from the page and check the status code.

**Comment by GlenDC** [comment]
Hmm okay, I suppose can close this one as a won't fix in that case?

**Comment by GlenDC** [comment]
> Chrome net proxy

You mean a proxy based on the chromiun network stack? Is that a plan you're already have in the make then, or just future dreams?

**Comment by blakebyrnes** [comment]
Yes, I do mean using the Chromium network stack. That's what we ultimately want to do, but it's just a future plan at the moment.

**Comment by GlenDC** [comment]
Exciting prospects :) I would say for now perhaps better hold it off, but we can keep the issue open in that case, at least that's my opinion.
